remote.go

```diff
@@ -1124,6 +1134,35 @@ func (wd *remoteWD) Screenshot() ([]byte, error) {
 	return ioutil.ReadAll(decoder)
 }
 
+func (wd *remoteWD) WaitWithTimeoutAndInterval(condition Condition, timeout time.Duration, interval time.Duration) (error) {
```

Comment:
Remove the parenthesis around the only return parameter. Parenthesis are only needed for multiple parameters.

Fix in the interface definition as well.